.github/workflows/border_router.yml

```diff
@@ -85,11 +87,24 @@ jobs:
       INTER_OP: 0
       INTER_OP_BBR: 0
       BORDER_ROUTING: ${{ matrix.border_routing }}
+      MAX_JOBS: 3
```

Comment:
OTBR tests seem pretty stable without this, do we really need this?

Reply:
Do you mean otbr tests in openthread?

Reply:
I mean the OTBR tests in openthread/ot-br-posix.

Reply:
Should be more reliable using `MAX_JOBS = 3`.